src/Microsoft.AspNet.TelemetryCorrelation/Microsoft.AspNet.TelemetryCorrelation.csproj

```diff
@@ -1,85 +1,19 @@
-﻿<Project ToolsVersion="15.0" Sdk="Microsoft.NET.Sdk">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory),Microsoft.AspNet.TelemetryCorrelation.sln))\tools\Common.props" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <ProjectGuid>{4C8E592C-C532-4CF2-80EF-3BDD0D788D12}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Microsoft.AspNet.TelemetryCorrelation</RootNamespace>
-    <AssemblyName>Microsoft.AspNet.TelemetryCorrelation</AssemblyName>
-    <TargetFrameworks>net45</TargetFrameworks>
-    <FileAlignment>512</FileAlignment>
-    <SignAssembly>true</SignAssembly>
-    <DocumentationFile>$(OutputPath)$(AssemblyName).xml</DocumentationFile>
-    <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\</SolutionDir>
-    <CodeAnalysisRuleSet>Microsoft.AspNet.TelemetryCorrelation.ruleset</CodeAnalysisRuleSet>
-    <RunCodeAnalysis>true</RunCodeAnalysis>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <DebugSymbols>true</DebugSymbols>
-    <DocumentationFile>$(OutputPath)/$(TargetFramework)/$(AssemblyName).xml</DocumentationFile>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(PublicRelease)' == 'True' ">
-    <DelaySign>true</DelaySign>
-    <AssemblyOriginatorKeyFile>$(RepositoryRoot)tools\35MSSharedLib1024.snk</AssemblyOriginatorKeyFile>
-    <DefineConstants>$(DefineConstants);PUBLIC_RELEASE</DefineConstants>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(PublicRelease)' != 'True' ">
-    <DelaySign>false</DelaySign>
-    <AssemblyOriginatorKeyFile>$(RepositoryRoot)tools\Debug.snk</AssemblyOriginatorKeyFile>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <DefineConstants>$(DefineConstants);DEBUG;TRACE</DefineConstants>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <DefineConstants>$(DefineConstants);TRACE</DefineConstants>
-  </PropertyGroup>
-  <PropertyGroup>
-    <Company>Microsoft Corporation</Company>
-    <!--https://docs.microsoft.com/en-us/nuget/schema/msbuild-targets-->
-    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
-    <IncludeSymbols>True</IncludeSymbols>
-    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-    <!--PackageOutputPath>Defined in Directory.Build.props</PackageOutputPath-->
-    <PackageId>Microsoft.AspNet.TelemetryCorrelation</PackageId>
-    <!--PackageVersion is defined in from VersionPrefix and VersionSuffix -->
-    <!--PackageVersion></PackageVersion-->
-    <Authors>Microsoft</Authors>
-    <Title>Microsoft Asp.Net telemetry correlation</Title>
+    <TargetFrameworks>net461</TargetFrameworks>

```

Comment:
@cijothomas @rajkumar-rangaraj do you see a problem here if we target `net461`? Previously the target was `<TargetFrameworks>net45</TargetFrameworks>`.

Reply:
this should be okay.
if anyone wants to use older than net461, then they can use the aspnet package.

Since Otel itself stopped support for older than net461, otel users shouldn't have issue either.